### PR TITLE
Enh - GitHub action build and push package to feed

### PR DESCRIPTION
**Pull request by domenicsim1**

### CODE REVIEW

File: build/Octopus.Server.Extensibility.JiraIntegration.nuspec [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/OctopusDeploy/JiraIntegration/blob/main/LICENSE.txt</licenseUrl>
+    <projectUrl>https://github.com/OctopusDeploy/JiraIntegration</projectUrl>
+    <repository type="git" url="https://github.com/OctopusDeploy/JiraIntegration" />
+    <iconUrl>https://i.octopusdeploy.com/resources/Avatar3_360.png</iconUrl>
+    <tags>automation deployment</tags>
+    <developmentDependency>true</developmentDependency>
+    <dependencies>
+      <group targetFramework="net5.0">
+        <dependency id="Novell.Directory.Ldap.NETStandard" version="3.6.0" exclude="Build,Analyzers" />
```
Comment (hnrkndrssn) [inline build/Octopus.Server.Extensibility.JiraIntegration.nuspec:23]: Not sure we need this one?

File: source/Server.Tests/JiraDeploymentData/JiraDeploymentDataSerializesCorrectly.cs [hunk 2]
```diff
@@ -37,7 +37,7 @@ public void DataSerializesCorrectly()
                             Url =
                                 "https://octopussample.com/app#/Spaces-1/projects/foo/releases/1.0.0/deployments/deployments-123",
                             Description = "Task Description",
-                            LastUpdated = new DateTimeOffset(new DateTime(2018, 10, 27)),
+                            LastUpdated = new DateTimeOffset(new DateTime(2018, 10, 27), TimeSpan.Zero),
```
Comment (hnrkndrssn) [inline source/Server.Tests/JiraDeploymentData/JiraDeploymentDataSerializesCorrectly.cs:40]: Nice!

**Review by hnrkndrssn — APPROVED** [review]
Just a question about a questionable dependency in the `nuspec` file, not sure if that's just a leftover from a different project that this has been copied from...

**Comment by hnrkndrssn** [comment]
Also make sure to update TeamCity to no longer try and run a build on this repo

File: build/Octopus.Server.Extensibility.JiraIntegration.nuspec (hunk 1 above)
Comment (domenicsim1) [reply build/Octopus.Server.Extensibility.JiraIntegration.nuspec]: I think you are right, I have removed it.

**Pull request merged**